fbw-a380x/src/systems/instruments/src/ND/instrument.tsx

```diff
@@ -244,7 +244,7 @@ class NDInstrument implements FsInstrument {
       <div ref={this.topRef}>
         <CdsDisplayUnit
           bus={this.bus}
-          displayUnitId={DisplayUnitID.CaptNd}
+          displayUnitId={this.efisSide == 'L' ? DisplayUnitID.CaptNd : DisplayUnitID.FoNd}
```

Comment:
Can we use `getDisplayIndex()` here too to be consistent?

Reply:
Fixed.